src/include/homestore/btree/btree.hpp

```diff
@@ -97,12 +97,14 @@ class Btree {
     // bool verify_tree(bool update_debug_bm) const;
     virtual std::pair< btree_status_t, uint64_t > destroy_btree(void* context);
     nlohmann::json get_status(int log_level) const;
-    void print_tree() const;
+
+    void print_tree(std::string file = "") const;
```

Comment:
Nitpick: Suggest to use const string reference to avoid copy on parameter passing

Reply:
Ack